test/org/opensha/commons/calc/magScalingRelations/magScalingRelImpl/TestTMG2017CruMagAreaRel.java

```diff
@@ -0,0 +1,58 @@
+package org.opensha.commons.calc.magScalingRelations.magScalingRelImpl;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+// tested against sceqsrc (https://github.com/thingbaijam/sceqsrc)
+
+public class TestTMG2017CruMagAreaRel {
+
+    @Test
+    public void testGetMedianMag() {
+        TMG2017CruMagAreaRel tmg = new TMG2017CruMagAreaRel();
+
+        assertTrue(Double.isNaN(tmg.getMedianMag(1, Double.NaN)));
+
+        // area 1
+        assertEquals(4.158246, tmg.getMedianMag(1, 90), 0.000001);
```

Comment:
in order for this to pass, you'd need to change the expected value to 4.158 or the tolerance to 0.001